### PR TITLE
Changed CSS values

### PR DESCRIPTION
**Pull request by shikhanansi**
Changes suggested in https://ephocks.atlassian.net/browse/DOC-291. @ephox-mogran, please review.

### CODE REVIEW

File: plugins/comments/comments_2.0.md [hunk 1]
```diff
@@ -86,7 +85,13 @@ The editor needs to be configured to highlight the commented text. The following
 ```js
 tinymce.init({
 ...
- content_style: '.mce-annotation { background: #fff0b7; } .tc-active-annotation {background: #ffe168; color: black; }',
+ content_style: '.tox-comments-visible span.tox-comment
```
Comment (ephox-mogran) [inline plugins/comments/comments_2.0.md]: This needs to be on one line, or use string concatenation

File: plugins/comments/comments_2.0.md [hunk 2]
```diff
@@ -54,15 +56,23 @@ tinymce.init({
...
 
-> Note: Currently, there are two toolbar buttons available:
+Use the following script to configure the Comments 2.0 toolbar button:
 
->* `addcomment` - Provides the ability to add comments.
->* `showcomments`- Provides the ability to display comments field for the selected text. It is a toggle button and is used to hide the comments sidebar as well.
+```js
+tinymce.init({
+ selector: "#textarea",
+ toolbar: 'bold italic underline insertfile | showcomment',
```
Comment (ephox-mogran) [inline plugins/comments/comments_2.0.md]: showcomments*

File: plugins/comments/comments_2.0.md (hunk 1 above)
Comment (shikhanansi) [reply plugins/comments/comments_2.0.md]: Fixed

File: plugins/comments/comments_2.0.md (hunk 2 above)
Comment (shikhanansi) [reply plugins/comments/comments_2.0.md]: Fixed

**Pull request merged**